learn/cmd/levy/main.go

```diff
@@ -70,7 +70,6 @@ type exportCmd struct {
 	PrivateKey        string `short:"k" help:"Secret private key to decrypt sealed answers." env:"EVY_LEARN_PRIVATE_KEY"`
 	WithAnswersMarked bool   `short:"m" help:"Include marked answers in HTML output. Cannot be used with export target answerkey."`
 	SelfContained     bool   `short:"c" help:"Ensure .html files are self-contained (embed CSS, JS, Favicon)." negatable:""`
-	SkipQuestions     bool   `short:"q" help:"Skip exporting individual questions, export full exercise only."`
```

Comment:
Why have you removed this? It does not seem related to the commit as described in the commit message.

Reply:
I guess I have removed this, IIRC, because I now see that I won't need it. I will have to export individual questions _always_ as htmlf. This will become clearer when I implement exercises/quiz/unittest question selection. I've updated the commit.